src/bridge.py

```diff
@@ -47,7 +47,7 @@ def start(version: str, proxy: bool = False) -> None:
     #   not to still think the bridge is running
     if proc is not None and not is_running():
         log("Bridge was probably killed by user manually, resetting local state")
-        stop(cleanup=True, proxy=proxy)
+        stop(proxy=proxy)
 
     if proc is not None:
         raise RuntimeError("Bridge is already running, not spawning a new one")
```

Comment:
Should we do the same with `start()`?

Reply:
Yes, we can remove raising this error - but then we need to agree what else to do in case bridge is being started, but it is already running. Most certainly two bridges cannot run at the same time, so we would either ignore the second one or kill the first one.

What is my worry, that without these RuntimeErrors, which communicate the situation to the client, the app will not be as responsive as possible (telling users they probably did something wrong - spawned a bridge two times etc.).

The biggest mystery right now is **Why does raising the RuntimeError in stop() kill the CI, when there is try-except in controller?**.

I would actually prefer to keep these RuntimeErrors, but we need to be really sure they do not cause issues in CI :)

Reply:
does it produces an error with a 1? that is probably why it fails if it does.

Reply:
I think I know now - the Suite Cypress tests are processing the response JSON from the websocket, and if there is `success: False`, they will fail the tests - more in #72

Reply:
Great find :+1:. Ok let's not solve something that does not pose a problem for us at the moment :). Let's keep the Rrror for `start`. Resolving.